proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md

```diff
@@ -0,0 +1,78 @@
+# Clarification of SDL App Icon display sequence and display order
+
+* Proposal: [SDL-NNNN](NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Policy Server ]
+
+## Introduction
+
+This proposal is to clarify the SDL App Icon display sequence and the display order of icons. By doing so, we can prevent unnecessary development for the HU developers and improve usabilty by enhancing the visuality for the users.
+
+## Motivation
+
+Currently, the SDL app icon is displayed on the SDL app selection (menu) screen, but the sequence to display the icon is not provided. Also, there is no explicit definition of the icon's display order and the rule for example grouping by app type, as a result, the display order of icons changes every launch. Incidentally, it is implemented on the program code of SDL Core by the following rule.
```

Comment:
>Also, there is no explicit definition of the icon's display order and the rule for example grouping by app type, as a result, the display order of icons changes every launch.

I'm having trouble understanding this sentence.  Can you please clarify and reword?

Reply:
@Akihiro-Miyazaki please address.

Reply:
Hi, @theresalech -san,
Sorry for late response. I will modify the PR file and make a commit it.
Thank you.